GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h

```diff
@@ -70,15 +70,17 @@ typedef NS_ERROR_ENUM(kGIDSignInErrorDomain, GIDSignInErrorCode) {
 
 /// Configures `GIDSignIn` for use.
 ///
+/// @param useDebugProvider Whether or not App Check should use the debug provider.
 /// @param completion A nullable callback block passing back any error arising from the
-/// configuration process if any exists.
+///     configuration process if any exists.
 ///
 /// Call this method on `GIDSignIn` prior to use and as early as possible. This method generates App
 /// Attest key IDs and the attestation object eagerly to minimize latency later on during the sign
 /// in or add scopes flows.
-- (void)configureWithCompletion:(nullable void (^)(NSError * _Nullable error))completion
+- (void)configureWithDebugProviderEnabled:(BOOL)useDebugProvider
+                               completion:(nullable void (^)(NSError * _Nullable error))completion
 API_AVAILABLE(ios(14))
-NS_SWIFT_NAME(configureWithCompletion(completion:));
+NS_SWIFT_NAME(configure(debugProviderEnabled:completion:));
```

Comment:
Now that we'll need an API key for the debug provider, but not for the standard App Attest flow, we may want to split this into two methods (`configureWithCompletion:` for default App Attest + a new one for debug).

cc: @weixifan

Reply:
Good callout. What is here on GitHub is not up-to-date with what I have locally. I will push up that change soon.